### PR TITLE
cariddi: init at 1.1

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Crawler for URLs and endpoints

https://github.com/edoardottt/cariddi

Related to #81418

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 139863` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>cariddi</li>
  </ul>
</details>

File: pkgs/tools/security/cariddi/default.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    sha256 = "0cwkycydwndda11m1jszbxchpqabrgspj17y2aj3y3m2x446h27z";
+  };
+
+  vendorSha256 = "0rmiya517i9s4l9nxzwly5vq8cqhhpq66rc7y4sapyaihx20ai3r";
+
+
+  meta = with lib; {
+    description = "Crawler for URLs and endpoints";
+    homepage = "https://github.com/edoardottt/cariddi";
+    license = with licenses; [ gpl3Plus ];
```
Comment (figsoda) [inline pkgs/tools/security/cariddi/default.nix:23]:
```suggestion
    license = licenses.gpl3Plus;
```

File: pkgs/tools/security/cariddi/default.nix [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    owner = "edoardottt";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0cwkycydwndda11m1jszbxchpqabrgspj17y2aj3y3m2x446h27z";
+  };
+
+  vendorSha256 = "0rmiya517i9s4l9nxzwly5vq8cqhhpq66rc7y4sapyaihx20ai3r";
+
+
+  meta = with lib; {
```
Comment (figsoda) [inline pkgs/tools/security/cariddi/default.nix:20]:
```suggestion
  vendorSha256 = "0rmiya517i9s4l9nxzwly5vq8cqhhpq66rc7y4sapyaihx20ai3r";

  meta = with lib; {
```

**Review by figsoda — CHANGES_REQUESTED** [review]

File: pkgs/tools/security/cariddi/default.nix (hunk 1 above)
Comment (fabaff) [reply pkgs/tools/security/cariddi/default.nix:23]: I don't get it what's the issue here? The license is [`GPLv3+`](https://github.com/edoardottt/cariddi/blob/main/output/txt.go#L6-L14).

Comment (SuperSandro2000) [reply pkgs/tools/security/cariddi/default.nix:22]: Either is fine. The with should be used when multiple licenses are used. If there is only one license it does not matter.

**Pull request merged**